xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -1648,7 +1709,7 @@ struct AppModel: ModelProtocol {
         // For now, we just sync everything on ready.
         return update(
             state: state,
-            action: .syncAll,
+            actions: [.syncAll, .checkRecoveryStatus],
```

Comment:
`.start` is slightly too early, it will trigger recovery mode on first-run of the app. We have to wait until we've either loaded the default sphere or created the default sphere before we can check the recovery status.